src/libs/elektra/keyname.c

```diff
@@ -1528,17 +1591,20 @@ static size_t keyAddBaseNameInternal (Key * key, const char * baseName)
  * E.g. if you add . it will be escaped:
  * @snippet testabi_key.c base1 add
  *
- * @see keySetBaseName() to set a base name
- * @see keySetName() to set a new name.
+ * @param key the Key to add the basename to
+ * @param baseName the string to append to the Key's name
  *
- * @param key the key object to work with
- * @param baseName the string to append to the name
- * @return the size in bytes of the new key name including the ending NULL
- * @retval -1 if the key had no name
+ * @return the size in bytes of the Key's new name including the NULL terminator
+ * @retval -1 if the Key has no name
  * @retval -1 on NULL pointers
- * @retval -1 if key was inserted to a keyset before
- * @retval -1 on allocation errors
+ * @retval -1 if Key was inserted to a KeySet before
```

Comment:
Maybe:
```suggestion
 * @retval -1 if Key was inserted into KeySet before
```

Reply:
fixed in https://github.com/ElektraInitiative/libelektra/pull/3795/commits/cea2c469decb39e99b3d71197f27ed40dede38d9